### PR TITLE
fix(file-upload): update style if label is absent [khcp-5654]

### PR DESCRIPTION
**Pull request by aanchalm01**
# Summary

## PR Checklist

* [ ] Does not introduce dependencies
* [ ] **Functional:** all changes do not break existing APIs and if so, bump major version.
* [ ] **Tests pass:** check the output of yarn test
* [ ] **Naming:** the files and the method and prop variables use the same naming conventions as other Kongponents
* [ ] **Framework style:** abides by the essential rules in Vue's style guide
* [ ] **Cleanliness:** does not have formatting issues, unused code (e.g., console.logs, debugger), or leftover comments
* [ ] **Docs:** includes a technically accurate README, uses JSDOC where appropriate

### CODE REVIEW

File: src/components/KFileUpload/KFileUpload.vue [hunk 1]
```diff
@@ -285,22 +286,38 @@ export default defineComponent({
 .k-file-upload {
   position: relative;
 
+  :deep(.k-input-label) {
+    pointer-events: none;
```
Comment (adamdehaven) [inline src/components/KFileUpload/KFileUpload.vue:290]: question: I don't think we want to block pointer-events on the label, is there a reason this is needed?

File: src/components/KFileUpload/KFileUpload.vue [hunk 2]
```diff
@@ -285,22 +286,38 @@ export default defineComponent({
...
+
+    .label-tooltip span.kong-icon.kong-icon-help {
+      pointer-events: visible;
```
Comment (adamdehaven) [inline src/components/KFileUpload/KFileUpload.vue:293]: If you keep this, instead of `visible` just set to `auto`

Comment (aanchalm01) [reply src/components/KFileUpload/KFileUpload.vue]: Updated.

File: src/components/KFileUpload/KFileUpload.vue (hunk 1 above)
Comment (adamdehaven) [reply src/components/KFileUpload/KFileUpload.vue:290]: The label can be clickable (this is standard web behavior) I think the issue was just the `.help` container was also clickable. 

I'd suggest removing the pointer-events rules you added, and then I think everything else LGTM

Comment (aanchalm01) [reply src/components/KFileUpload/KFileUpload.vue]: Updated.

**Review by adamdehaven — APPROVED** [review]
LGTM

**Pull request merged**